packages/opentelemetry-tracing/src/BasicTracerProvider.ts

```diff
@@ -32,10 +35,25 @@ import { MultiSpanProcessor } from './MultiSpanProcessor';
 import { NoopSpanProcessor } from './NoopSpanProcessor';
 import { SDKRegistrationConfig, TracerConfig } from './types';
 import merge = require('lodash.merge');
+
+export type PROPAGATOR_FACTORY = () => TextMapPropagator;
+
 /**
  * This class represents a basic tracer provider which platform libraries can extend
  */
 export class BasicTracerProvider implements TracerProvider {
+  protected static readonly _registeredPropagators = new Map<
+    string,
+    PROPAGATOR_FACTORY
+  >([
+    ['tracecontext', () => new HttpTraceContext()],
+    ['baggage', () => new HttpBaggage()],
+  ]);
+
+  static registerPropagator(name: string, factory: PROPAGATOR_FACTORY): void {
```

Comment:
I'm still not sure why we have this registration. Since we depend directly on b3 and jaeger propagators already anyway there's no reason I can see not to hardcode in some map from ENV name to propagator. Third party propagators would already require code changes to call this register method so there is no convenience advantage I can see.

Reply:
Alright, unless any vendors come here and make a case for `registerPropagator` I'll remove it and fix the tests tomorrow.

Reply:
I think we can remove it then :)

Reply:
done